tests/flow.html

```diff
@@ -93,36 +160,52 @@ <h3>
       let loadVexFlow;
       const isFileProtocol = location.protocol === 'file:';
       if (useESM && isFileProtocol) {
-        console.warn('ES modules require a web server for testing!');
+        prompt(
+          'ES modules require a web server for testing!\n\nTry `npx http-server` and browse to http://127.0.0.1:8080/tests/flow.html?esm=true\n\nWe will fall back to CJS for now.',
+          'npx http-server'
+        );
       }
       if (useESM && !isFileProtocol) {
-        console.log('LOADING ESM: VexFlow >= 4.0.0');
-        // This ESM branch only works if flow.html is accessed from a web server
+        console.log('LOADING ESM');
+        // ESM version only works if flow.html is accessed from a web server
         // (e.g., `npx http-server` and browse to http://127.0.0.1:8080/tests/flow.html).
         loadVexFlow = async () => {
           const module = await import('../build/esm/entry/vexflow-debug-with-tests.js');
           window.Vex = module.Vex;
+          return Vex.Flow;
         };
       } else {
-        console.log('LOADING CJS: VexFlow >= 4.0.0');
+        console.log('LOADING CJS');
         // When loading version >= 4.0.0, only load vexflow-debug-with-tests.js
-        loadVexFlow = () => loadScript(vexPlusTestsURL);
+        loadVexFlow = async () => {
+          await loadScript(vexPlusTestsURL);
+          return Vex.Flow;
+        };
       }
 
-      await loadVexFlow();
+      // Only proceed after all fonts have loaded.
+      const fontFaceSet = await document.fonts.ready;
```

Comment:
This is one approach to waiting until all fonts are loaded before starting VexFlow.

The other way is to call `loadWebFonts(...)`, which will load a woff2 from a CDN.